retroarch-standalone-service/retroarch-standalone.xinitrc

```diff
@@ -6,10 +6,18 @@ xset -dpms         # disable DPMS (Energy Star) features.
 xset s noblank     # don't blank the video device
 
 # Hide mouse cursor
-unclutter -display :0.0 -noevents -grab -root -reset &
+unclutter -display :0.0 -root &
 
 # Clean old debug logs
 rm /home/retroarch/debug.log 2>&1 >&/dev/null
 
 # Autostart Retroarch
-exec /usr/bin/retroarch --appendconfig=/etc/retroarch-standalone.cfg -v 2>&1 >&/home/retroarch/debug.log
+exit_code=1
+while [ "$exit_code" -ne "0" ]
+do
+  /usr/bin/retroarch --appendconfig=/etc/retroarch-standalone.cfg -v --log-file /home/retroarch/debug.log
```

Comment:
There's a report from a user saying that now RetroArch has built-in logging interface. If it hooks up into journald we could remove the redirect to the log-file :) What do you think?

Reply:
I'm not aware of anything builtin to RetroArch for logging to journald. There is a config setting `log_to_file` but I am not having any luck when testing it.

Reply:
Let's keep it as it is then :) thanks